<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesNavigator</class>
 <widget class="QWidget" name="FilesNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Работа с данными</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>356</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>356</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>386</width>
     <height>600</height>
    </size>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Импорт данных</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Экспорт выбранного файла</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>330</width>
     <height>286</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>270</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QWidget" name="previewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>240</height>
     </size>
    </property>
    <widget class="QLabel" name="lbl_v">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_p">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Предпросмотр</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
